src/PrestaShopBundle/Form/Admin/Type/TypeaheadProductCollectionType.php

```diff
@@ -89,7 +89,7 @@ public function buildView(FormView $view, FormInterface $form, array $options)
                         $product = $this->productAdapter->getProduct($id);
                         $collection[] = array(
                             'id' => $id,
-                            'name' => $product->name[1].' (ref:'.$product->reference.')',
+                            'name' => $product->name[(int)\Context::getContext->language->id].' (ref:'.$product->reference.')',
```

Comment:
Unfortunately, we avoid calls to the legacy code from the `src/` folder.
We have two options here:
* Using `reset($product->name)` to get the first language available in the array
* Inject the service legacy context, call its method `getLanguages(true)` and apply a `reset()` on the restart to get the default language.